cover_art_api.py

```diff
@@ -1,35 +1,38 @@
+from io import BytesIO
 from PIL import Image
-import shutil
 import requests
+import shutil
 import os
 import glob
 from pprint import pprint
 
 url = f"http://ws.audioscrobbler.com/2.0"
-#key = os.environ.get("COVER_KEY") # "2066002fabe29cf09d7ed5a00c804ac6"
+# key = os.environ.get("COVER_KEY") # "2066002fabe29cf09d7ed5a00c804ac6"
 key = "2066002fabe29cf09d7ed5a00c804ac6"
```

Comment:
We will need to get this key removed.

Reply:
Yes - That's still an issue. Thanks for reminding! I know you two got yours to work without issue so let's review together.